pkg/code-generator/codegen/templates/snapshot_emitter_test_template.go

```diff
@@ -70,11 +73,14 @@ var _ = Describe("{{ upper_camel .Project.ProjectConfig.Version }}Emitter", func
 		cfg, err = clientcmd.BuildConfigFromFlags("", kubeconfigPath)
 		Expect(err).NotTo(HaveOccurred())
 
+{{- if $needs_crd_client }}
+
 		cache := kuberc.NewKubeCache()
+{{- end}}
 
-{{- if $needs_kube_client }}
-var kube kubernetes.Interface
-{{- end }}
+{{- if $needs_standard_client }}
+		var kube kubernetes.Interface
+{{- end}}
```

Comment:
Instead of declaring it here, you could use `:=` instead of `=` on line 95. It's used only there.

Reply:
i can't because if there are multiple resources that use it, we'll have a redeclare error:
```
kube, err := ...
....
kube, err := ... < -- error here
```